src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -192,12 +233,14 @@ public String checkInjection() {
   }
 
   // put all the key words together
-  // use a HashSet to remove duplicates
+  // use a LinkedHashSet to remove duplicates but maintain order
   public Set<String> getKeyWords() {
     try {
-      Set<String> keyWords = new HashSet<String>();
-      keyWords.addAll(getEvents());
+      Set<String> keyWords = new LinkedHashSet<String>();
```

Comment:
Since the keyword set is starting to get bigger, it would be nice to classify this data. I think instead of LinkedHashSet we can start to use LinkedHashMap. Probably we can do this after the MVP presentation, because this change will also affect my feature.

Reply:
Yes, I agree.